kubejs/server_scripts/End_Game.js

```diff
@@ -84,34 +85,108 @@ ServerEvents.recipes(event => {
             },
             "X": {
                 "item": "gtceu:energetic_alloy_ingot"
+            },
+            "Y": {
+                "item": "gtceu:dark_soularium_ingot"
             }
         },
         "result": {
             "item": "kubejs:exotic_materials_catalyst"
         }
     })
 
+    // Blacklight
+    event.remove({ output: 'gtceu:blacklight' })
+    event.shaped(
+        'gtceu:blacklight', [
+            'BPB',
+            ' S ',
+            'CPW'
+        ], {
+            B: 'gtceu:tungsten_carbide_screw',
+            C: '#gtceu:circuits/iv',
+            P: 'gtceu:tungsten_carbide_plate',
+            S: 'gtceu:hssg_spring',
+            W: 'gtceu:platinum_single_cable'
+        }
+    )
```

Comment:
General code style advice: If you know the recipe id you can override the recipe by adding `.id(oldRecipeID` to the end of the new one. WAY faster than removing by output